src/styles/blocks/catalog-page.scss

```diff
@@ -0,0 +1,38 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  transition: all .3s;
```

Comment:
it's not a good idea setting these properties with * selector.
use it only for box-sizing. reset magings for some elements you need for instance:

```scss
body,
p,
h2,
// other selectors {
  margin: 0;
}
``` 

and set transition for those elements you need